asllib/Parser0.mly

```diff
@@ -453,7 +455,16 @@ let getter_decl ==
         and subprogram_type = ST_EmptyGetter
         and recurse_limit = None
         and parameters = [] in
-        D_Func { name; args; return_type; body; parameters; subprogram_type; recurse_limit }
+        D_Func {
+          name;
+          args;
+          return_type;
+          body;
+          parameters;
+          subprogram_type;
+          recurse_limit;
+          builtin = false;
```

Comment:
is this meant to identify whether something is part of the blessed functions or not?

Reply:
Yes exactly! The flag is set to `true` for stdlib/primitives in `builder.ml`